js/apps/account-ui/src/routes.tsx

```diff
@@ -59,7 +59,7 @@ export const PersonalInfoRoute: IndexRouteObject = {
 };
 
 export const RootRoute: RouteObject = {
-  path: new URL(environment.baseUrl).pathname,
+  path: decodeURI(new URL(environment.baseUrl).pathname),
```

Comment:
There should be no need to decode this I think, as `URL` should already take care of it. But if you do you want to likely go for [`decodeURIComponent()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent) since this is not decoding a whole URL, but only the path, there are [some differences](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURI#decodeuri_vs._decodeuricomponent).

Reply:
The decode is necessary, if not I have an error 404 with route (I detected this today, previously account v2 worked flawlessly). It seems that the router uses a decoded path. I  will change `decodeURI` with `decodeURIComponent`.

Reply:
Done now!